setup.py

```diff
@@ -115,7 +115,7 @@ def find_version(*file_paths):
     "docs": [],
     "tf": [
         "tensorflow~=2.8.2",
-        "numpy~=1.22.1"
+        "numpy~=1.23.1"
```

Comment:
Could you remove the dependency on the **exact** version of numpy from NNCF? I suggest to include numpy as common dependency of NNCF without version specification.

cc' @vshampor, @andrey-churkin, @MaximProshin, @AlexKoff88

Reply:
Good point, I'll move it to common dependency with range and test it on the CI and local server.